src/hotspot/share/gc/shenandoah/c2/shenandoahSupport.cpp

```diff
@@ -387,6 +387,9 @@ void ShenandoahBarrierC2Support::verify(RootNode* root) {
           verify_type t;
         } args[6];
       } calls[] = {
+        "arraysort_stub",
+        { { TypeFunc::Parms, ShenandoahLoad },   { TypeFunc::Parms+1, ShenandoahStore },  { -1, ShenandoahNone },
```

Comment:
Only the first parameter is array.

Reply:
Please see the fixed signature in the latest commit.